src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -34,6 +38,14 @@ def __init__(
             if the plane is a combined plane as in the Suite2p pipeline
         plane_no: int
             the plane for which to extract segmentation for.
+        search_plane_subdirectory: bool
+            If True, will search for files in 'folder_path/plane{plane_no}',
+             else will search for files in 'folder_path'.
```

Comment:
After discussing it a bit, we don't think it's a good idea to support this level of implicit behavior as it makes the reader more complex than needed; in particular we find it simplest if reader classes take a single input that always behaves in a single explicit way

This minimizes the number of use cases, tests, and cross-argument conditions that need to be tested and supported

So within the current framework, we want to support the pattern where `Suite2P` as a format takes a (i) top-level `folder_path` that contains subfolders for each plane, and specifies which plane to load data for using the (ii) `plane_no` argument

Now, there's a lot we would like to improve on this in the long-term... namely, fully explicit control over which streams are detected (via a class method) and loaded (via user selection of streams from that detection) so that this kind of control is even more explicit for users; but until we get the bandwidth to do a wider refactor of that over the repo, we'll have to stay within the current paradigm

Reply:
Sounds good